stl/inc/chrono

```diff
@@ -5486,34 +5497,79 @@ struct formatter<_CHRONO day, _CharT> {
         }
 
         for (const auto& _Spec : _List) {
-            if (_Spec._Type != '\0' && _Spec._Type != 'd' && _Spec._Type != 'e') {
-                _THROW(format_error("Invalid type for type day."));
-            }
-            if (_Spec._Modifier != '\0' && _Spec._Modifier != 'O') {
-                _THROW(format_error("Invalid modifier for type day."));
+            if (_Spec._Type != '\0' && _RANGES find(_Legal_types, _Spec._Type) != _Legal_types.end()) {

```

Comment:
I believe that the iterator comparison here is reversed. If the iterator returned by `find()` is equal to `end()`, then it wasn't found among the valid types, and we should fail.